src/filters/EnergyFilter.C

```diff
@@ -0,0 +1,877 @@
+    4.5709e5,  4.7863e5,  5.0119e5,  5.2481e5,  5.4954e5,  5.7544e5,  6.0256e5,  6.3096e5,
+    6.6069e5,  6.9183e5,  7.2444e5,  7.5858e5,  7.9433e5,  8.3176e5,  8.7096e5,  9.1201e5,
+    9.5499e5,  1.0000e6,  1.0471e6,  1.0965e6,  1.1482e6,  1.2023e6,  1.2589e6,  1.3183e6,
+    1.3804e6,  1.4454e6,  1.5136e6,  1.5849e6,  1.6596e6,  1.7378e6,  1.8197e6,  1.9055e6,
+    1.9953e6,  2.0893e6,  2.1878e6,  2.2909e6,  2.3988e6,  2.5119e6,  2.6303e6,  2.7542e6,
+    2.8840e6,  3.0200e6,  3.1623e6,  3.3113e6,  3.4674e6,  3.6308e6,  3.8019e6,  3.9811e6,
+    4.1687e6,  4.3652e6,  4.5709e6,  4.7863e6,  5.0000e6,  5.2000e6,  5.4000e6,  5.6000e6,
+    5.8000e6,  6.0000e6,  6.2000e6,  6.4000e6,  6.6000e6,  6.8000e6,  7.0000e6,  7.2000e6,
+    7.4000e6,  7.6000e6,  7.8000e6,  8.0000e6,  8.2000e6,  8.4000e6,  8.6000e6,  8.8000e6,
+    9.0000e6,  9.2000e6,  9.4000e6,  9.6000e6,  9.8000e6,  1.0000e7,  1.0200e7,  1.0400e7,
+    1.0600e7,  1.0800e7,  1.1000e7,  1.1200e7,  1.1400e7,  1.1600e7,  1.1800e7,  1.2000e7,
+    1.2200e7,  1.2400e7,  1.2600e7,  1.2800e7,  1.3000e7,  1.3200e7,  1.3400e7,  1.3600e7,
+    1.3800e7,  1.4000e7,  1.4200e7,  1.4400e7,  1.4600e7,  1.4800e7,  1.5000e7,  1.5200e7,
+    1.5400e7,  1.5600e7,  1.5800e7,  1.6000e7,  1.6200e7,  1.6400e7,  1.6600e7,  1.6800e7,
+    1.7000e7,  1.7200e7,  1.7400e7,  1.7600e7,  1.7800e7,  1.8000e7,  1.8200e7,  1.8400e7,
+    1.8600e7,  1.8800e7,  1.9000e7,  1.9200e7,  1.9400e7,  1.9600e7,  1.9800e7,  2.0000e7,
+    2.0200e7,  2.0400e7,  2.0600e7,  2.0800e7,  2.1000e7,  2.1200e7,  2.1400e7,  2.1600e7,
+    2.1800e7,  2.2000e7,  2.2200e7,  2.2400e7,  2.2600e7,  2.2800e7,  2.3000e7,  2.3200e7,
+    2.3400e7,  2.3600e7,  2.3800e7,  2.4000e7,  2.4200e7,  2.4400e7,  2.4600e7,  2.4800e7,
+    2.5000e7,  2.5200e7,  2.5400e7,  2.5600e7,  2.5800e7,  2.6000e7,  2.6200e7,  2.6400e7,
+    2.6600e7,  2.6800e7,  2.7000e7,  2.7200e7,  2.7400e7,  2.7600e7,  2.7800e7,  2.8000e7,
+    2.8200e7,  2.8400e7,  2.8600e7,  2.8800e7,  2.9000e7,  2.9200e7,  2.9400e7,  2.9600e7,
+    2.9800e7,  3.0000e7,  3.0200e7,  3.1623e7,  3.3113e7,  3.4674e7,  3.6308e7,  3.8019e7,
+    3.9811e7,  4.1687e7,  4.3652e7,  4.5709e7,  4.7863e7,  5.0119e7,  5.2481e7,  5.4954e7,
+    5.7544e7,  6.0256e7,  6.3096e7,  6.6069e7,  6.9183e7,  7.2444e7,  7.5858e7,  7.9433e7,
+    8.3176e7,  8.7096e7,  9.1201e7,  9.5499e7,  1.0000e8,  1.0471e8,  1.0965e8,  1.1482e8,
+    1.2023e8,  1.2589e8,  1.3183e8,  1.3804e8,  1.4454e8,  1.5136e8,  1.5849e8,  1.6596e8,
+    1.7378e8,  1.8197e8,  1.9055e8,  1.9953e8,  2.0893e8,  2.1878e8,  2.2909e8,  2.3988e8,
+    2.5119e8,  2.6303e8,  2.7542e8,  2.8840e8,  3.0200e8,  3.1623e8,  3.3113e8,  3.4674e8,
+    3.6308e8,  3.8019e8,  3.9811e8,  4.1687e8,  4.3652e8,  4.5709e8,  4.7863e8,  5.0119e8,
+    5.2481e8,  5.4954e8,  5.7544e8,  6.0256e8,  6.3096e8,  6.6069e8,  6.9183e8,  7.2444e8,
+    7.5858e8,  7.9433e8,  8.3176e8,  8.7096e8,  9.1201e8,  9.5499e8,  1.e9};
+
+constexpr double S_ECCO_1968[]{
```

Comment:
This is really minor, but I'm slightly in favor of moving these group structures into a header file. Maybe we'll want to include them elsewhere in the code base.